doc/changelogs/CHANGELOG_V22.md

```diff
@@ -57,6 +58,379 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="22.15.0"></a>
+
+## 2025-04-16, Version 22.15.0 'Jod' (LTS), @RafaelGSS
+
+### Notable Changes
+
+* \[[`3c88f3938b`](https://github.com/nodejs/node/commit/3c88f3938b)] - **(SEMVER-MINOR)** **assert**: implement partial error comparison (Ruben Bridgewater) [#57370](https://github.com/nodejs/node/pull/57370)
+* \[[`db19a3f9fc`](https://github.com/nodejs/node/commit/db19a3f9fc)] - **(SEMVER-MINOR)** **assert**: improve partialDeepStrictEqual (Ruben Bridgewater) [#57370](https://github.com/nodejs/node/pull/57370)
+* \[[`b0c162dbe3`](https://github.com/nodejs/node/commit/b0c162dbe3)] - **(SEMVER-MINOR)** **assert,util**: improve performance (Ruben Bridgewater) [#57370](https://github.com/nodejs/node/pull/57370)
+* \[[`159b9cf79b`](https://github.com/nodejs/node/commit/159b9cf79b)] - **(SEMVER-MINOR)** **benchmark**: adjust assert runtimes (Ruben Bridgewater) [#57370](https://github.com/nodejs/node/pull/57370)
+* \[[`073b15d52d`](https://github.com/nodejs/node/commit/073b15d52d)] - **(SEMVER-MINOR)** **benchmark**: skip running some assert benchmarks by default (Ruben Bridgewater) [#57370](https://github.com/nodejs/node/pull/57370)
+* \[[`1922cb1042`](https://github.com/nodejs/node/commit/1922cb1042)] - **(SEMVER-MINOR)** **benchmark**: add assert partialDeepStrictEqual benchmark (Ruben Bridgewater) [#57370](https://github.com/nodejs/node/pull/57370)
```

Comment:
I think this is not semver minor nor really notable

Reply:
This is automatically generated by changelog-maker. The PR where this commit belongs contains a semver-minor.